pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -504,3 +527,12 @@ func (p *staticPolicy) generateCPUTopologyHints(availableCPUs cpuset.CPUSet, reu
 
 	return hints
 }
+
+func isSMTAwarePolicyOptionEnabled(policyOptions []string) bool {
+	for _, option := range policyOptions {
+		if option == SMTAwarePolicyOption {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
This would likely either become code that sits directly inside the `Options` constructor, or a private function hanging off the `Options` type.

Reply:
ack, I think we should start putting this logic in the constructor.